src/sidebar/components/slider.js

```diff
@@ -11,6 +17,7 @@ import propTypes from 'prop-types';
  * order.
  *
  * Currently the only reveal/expand direction supported is top-down.
+ * @param {SliderProps} props
```

Comment:
Can you leave an empty line between the description and the `@param` block of a function's docs.